openslides_backend/action/mixins/import_mixins.py

```diff
@@ -66,8 +66,11 @@ def __init__(
         self.name_to_ids: Dict[SearchFieldType, List[Dict[str, Any]]] = defaultdict(
             list
         )
-        for name, _ in name_entries:
-            self.name_to_ids[name] = []
+        for name, name_entry in name_entries:
+            if name in self.name_to_ids:
+                self.name_to_ids[name].append(name_entry)
+            else:
+                self.name_to_ids[name] = []
```

Comment:
I don't like init a defaultdict with empty lists. I don't like the type of name_to_ids. I would expect
`Dict[str, List[int]]`.
Yes, the tests works. As far as I understand the code, it seems to be correct and I don't see the 
problems with lines 107-116. So I approve it.